README.md

```diff
@@ -442,6 +442,26 @@ Required and allowed options (minimal example):
 ```
 The dropdown will start from `hours_from` and will offer options until `hours_to`, with a stepsize of `hour_step`.
 
+### Type: Unsubscribe
+Including an unsubscribe question type will display a card that allows the user to unsubscribe from the protocol. Typically, you want only one `unsubscribe` question in your questionnaire, as the first item in the questionnaire. You may want to control its visibility by specifying a `show_after` property.
+
+Required and allowed options (minimal example):
+
+```ruby
+[{
+  type: :unsubscribe,
+  title: 'Klaar met dit schooljaar?',
+  content: 'Ben je klaar met dit schooljaar? Klik dan op de knop \'Onderzoek afronden\' om het onderzoek te voltooien.',
+  button_text: 'Onderzoek afronden',
+  show_after: Time.new(2018, 6, 15).in_time_zone
+}]
+```
+
+Required properties for an unsubscribe `question` type are `title`, `content`, and `button_text`.
```

Comment:
`button_text` is niet required toch?

Reply:
Idd, thanks. Dit had ik later gefixt, vandaar.